certbot/docs/using.rst

```diff
@@ -709,12 +709,24 @@ Where are my certificates?
 ==========================
 
 All generated keys and issued certificates can be found in
-``/etc/letsencrypt/live/$domain``. In the case of creating a SAN certificate
-with multiple alternative names, ``$domain`` is the first domain passed in
-via -d parameter. Rather than copying, please point
-your (web) server configuration directly to those files (or create
-symlinks). During the renewal_, ``/etc/letsencrypt/live`` is updated
-with the latest necessary files.
+``/etc/letsencrypt/live/$domain``, where ``$domain`` is the certificate
+name (see the note below). Rather than copying, please point your (web)
+server configuration directly to those files (or create symlinks).
+During the renewal_, ``/etc/letsencrypt/live`` is updated with the latest
+necessary files.
+
+.. note::
+  The certificate name ``$domain`` used in the path
+  ``/etc/letsencrypt/live/$domain``:
+
```

Comment:
It might be easier to read this list if it used only sentence fragments or full sentences, not a mixture of both. I don't see a way to easily do that though, so I'm fine to keep it as-is.

Reply:
What about this?

```
.. note::
  The certificate name ``$domain`` used in the path ``/etc/letsencrypt/live/$domain``
  follows the following convention:

  * it is the name given to ``--cert-name``,
  * if ``--cert-name`` is not set by the user it is the first domain given to
    ``--domains``,
  * if the first domain is a wildcard domain (eg. ``*.example.com``) the
    certificate name will be ``example.com``,
  * if a name collision would occur with a certificate already named ``example.com``,
    the new certificate name will be constructed using a numerical sequence
    as ``example.com-001``.
```

Reply:
Yes! That's great.

Reply:
I updated the PR with this new proposition.